### ISSUE TITLE
TileExporter requestedPixelSize method gives incorrect results

### ISSUE THREAD

**Issue by juliangilbey** [qupath/qupath#648]
In the file `qupath-core/src/main/java/qupath/lib/images/writers/TileExporter.java` the definition of the `requestedPixelSize` method has the fraction the wrong way up; it should instead read:

```java
	public TileExporter requestedPixelSize(double pixelSize) {
		this.downsample = pixelSize / server.getPixelCalibration().getAveragedPixelSize().doubleValue();
		return this;
	}
```

Otherwise, if the requested pixel size is 5.0, say, and the average pixel size is 0.25, the downsample value is set to 1/20 instead of the correct 20.

This affects the current GitHub version of QuPath (at commit 3ca92f9026c9c32bcfb28de419673d2beac49c4a).

**Comment by petebankhead** [comment]
Good catch, thanks. Looks like this was a late and not-thought-through late addition - it should be ok to calculate the downsample and pass it directly, as [in the examples in the docs](https://qupath.readthedocs.io/en/latest/docs/advanced/exporting_images.html#tile-exporter).

Will fix this for the next release.

**Comment by juliangilbey** [comment]
Indeed! I was reading the source code to try to create a specialised version of TileExporter and spotted this.

**Comment by petebankhead** [comment]
Fixed in https://github.com/qupath/qupath/pull/680 - will be included in v0.3.0.
